test/e2e/info_test.go

```diff
@@ -49,4 +54,32 @@ var _ = Describe("Podman Info", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(session.IsJSONOutputValid()).To(BeTrue())
 	})
+	It("podman info rootless storage path", func() {
+		if !rootless.IsRootless() {
+			Skip("test rootless_storage_path for rootless")
+		}
+		SkipIfRemote()
+		configPath := filepath.Join(os.Getenv("HOME"), ".config", "containers", "storage.conf")
+		err := os.RemoveAll(filepath.Dir(configPath))
```

Comment:
This makes me uncomfortable. I think these tests run only in CI, and can't be run by developers, but if that ever changes this could be catastrophic for regular users: it would destroy their existing storage.conf.

Did you consider something such as this instead?
```
    preserved_home := getenv("HOME")
    (add magic fail-safe such that HOME is restored on exit or failure)
    setenv("HOME", random-tmpdir)
    ...then create the storage.conf, etc
```

Reply:
Lets try that.

Reply:
fixed